plugins/CustomDimensions/vue/src/List/List.vue

```diff
@@ -0,0 +1,176 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div>
+    <div v-content-intro>
+      <h2>
+        <EnrichedHeadline>{{ translate('CustomDimensions_CustomDimensions') }}</EnrichedHeadline>
+      </h2>
+      <p v-html="$sanitize(contentIntroText)"></p>
+      <p v-show="isLoading">
+        <span class="loadingPiwik">
+          <img src="plugins/Morpheus/images/loading-blue.gif" />
+          {{ translate('General_LoadingData') }}
+        </span>
+      </p>
+    </div>
+    <div
+      v-show="!isLoading"
+      v-for="scope in availableScopes"
+      :key="scope.value"
+    >
+      <ContentBlock :content-title="`${scope.name} Dimensions`">
+        <p>
+          {{ translate(`CustomDimensions_ScopeDescription${ucfirst(scope.value)}`) }}
+          {{ translate(`CustomDimensions_ScopeDescription${ucfirst(scope.value)}MoreInfo`) }}
+        </p>
+        <table v-content-table>
+          <thead>
+            <tr>
+              <th class="index">{{ translate('General_Id') }}</th>
+              <th class="name">{{ translate('General_Name') }}</th>
+              <th
+                class="extractions"
+                v-show="scope.supportsExtractions"
+              >{{ translate('CustomDimensions_Extractions') }}</th>
+              <th class="active">{{ translate('CorePluginsAdmin_Active') }}</th>
+              <th class="action">{{ translate('General_Action') }}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-show="scope.numSlotsUsed === 0 && !isLoading">
+              <td colspan="5">{{ translate('CustomDimensions_NoCustomDimensionConfigured') }}</td>
+            </tr>
+            <tr
+              class="customdimension"
+              v-for="customDimension in sortedCustomDimensionsByScope[scope.value]"
+              :class="customDimension.idcustomdimension"
+              :key="customDimension.idcustomdimension"
+            >
+              <td class="index">{{ customDimension.idcustomdimension }}</td>
+              <td class="name">{{ customDimension.name }}</td>
+              <td
+                class="extractions"
+                v-show="scope.supportsExtractions"
+              >
+                <span :class="{'icon-ok': customDimension.extractions[0]?.pattern}" />
+              </td>
+              <td class="active"><span :class="{'icon-ok': customDimension.active}" /></td>
+              <td class="action">
+                <a
+                  class="table-action icon-edit"
+                  :href="`#?idDimension=${customDimension.idcustomdimension}&scope=${scope.value}`"
+                />
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <div class="tableActionBar">
+          <a
+            value
+            :class="{'disabled': !scope.numSlotsLeft}"
```

Comment:
That doesn't seem to have any effect at all. Might have allready been the case before. But if no slots are left, the create button is still displayed the same way and a click opens the form (which shows an error after submitting)
Does that maybe used to be a button element? The empty `value` attribute might indicate that :thinking:

Reply:
It's a link in list.directive.html, probably because the inner HTML has an icon so it can't be put in a value attribute. The handling of the `disabled` class on links is broken in the existing customdimensions template. This sounds like a candidate for a new issue.

Reply:
ok. I'll prepare a PR to fix that once this one was merged.